src/render.jl

```diff
@@ -56,17 +56,6 @@ function save(file::String, modulename::Module; args...)
     save(file, mime, documentation(modulename), config)
 end
 
-type Entries
-    entries::Vector{@compat(Tuple{Module, Any, AbstractEntry})}
-end
-Entries() = Entries(@compat(Tuple{Module, Any, AbstractEntry})[])
-
-function push!(ents::Entries, modulename::Module, obj, ent::AbstractEntry)
-    push!(ents.entries, (modulename, obj, ent))
-end
```

Comment:
@peter1000 what's the reasoning behind splitting `Entries` into `EntriesHtml` and `EntriesMD`? They look the same to me.